docs/hadoop-catalog-with-s3.md

```diff
@@ -0,0 +1,451 @@
+---
+title: "Hadoop catalog with S3"
+slug: /hadoop-catalog-with-s3
+date: 2025-01-03
+keyword: Hadoop catalog S3
+license: "This software is licensed under the Apache License version 2."
+---
+
+This document describes how to configure a Hadoop catalog with S3. 
+
+## Prerequisites
+
+In order to create a Hadoop catalog with S3, you need to place [`gravitino-aws-bundle-${gravitino-version}.jar`](https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-aws-bundle) in Gravitino Hadoop classpath located 
+at `${HADOOP_HOME}/share/hadoop/common/lib/`. After that, start Gravitino server with the following command:
+
+```bash
+$ bin/gravitino-server.sh start
+```
+
+## Create a Hadoop Catalog with S3
```

Comment:
I prefer to add credential vending to the examples by default.

Reply:
Have you added any detailed examples of credential vending in the hadoop-catalog.md?  so I can create link there.

Reply:
I didn't provide detailed example yet,  you could refer to the credential-vending document for detailed properties.